src/genepsi3d.f90

```diff
@@ -131,7 +132,7 @@ subroutine genepsi3d(ep1)
     logical :: dir_exists
     real(mytype),dimension(xsize(1),xsize(2),xsize(3)) :: ep1
     !
-    if (nrank==0) then
+    if (nrank==0.and.mod(itime,ilist)==0) then
```

Comment:
Is this called in the time loop?

Reply:
Hi Paul yes this is just to avoid the print at every iteration in the moving geo. The ALM removal from .i3d for the wind turbine have been added added by Nikos since now these two parameters should not be anymore in the inout file.